### PR TITLE
Feature/manpages

### PR DESCRIPTION
**Pull request by neolynx**
See: http://molior.info/doc/molior-tools

### CODE REVIEW

**Comment by randombenj** [comment]
Awesome! :confetti_ball:

File: doc/molior-create.1.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+NAME
+ molior-create - create molior top level git repo from templates
+
+SYNOPSIS
+ molior-create PROJECT VERSION
+
+DESCRIPTION
+ molior-create updates debian/changelog with sign-off commit messages and creates a release git tag.
```
Comment (randombenj) [inline doc/molior-create.1.txt]: Copy paste error?

**Review by randombenj — APPROVED** [review]

File: doc/molior-create.1.txt (hunk 1 above)
Comment (neolynx) [reply doc/molior-create.1.txt]: good catch, thanks ! :)

**Review by szakalboss — APPROVED** [review]

File: doc/molior-deploy.8.txt [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  -p PRJSRC_FILE    Do not download project source from molior sever, use filecontents
+  -m MOLIOR_SERVER  Molior server to use (Default in ~/.moliorrc)
+  -a APTLY_SERVER   Aptly server to use (Default in ~/.moliorrc)
+  -A                Override project apt sources with APTLY_SERVER
+  -V PKG_VERSION    Pin pkg version to install. Dangerous, use with care!
+  -O                Automatically find top level package in the available projectversions
+  -o OVERLAY_BASE   Specify overlay base and fetch top level package from there
+  -c                Cleanup on error (Default: ask)
+  -f                Build all variants (Default: ask)
+  -F OUTPUT_FILE    Custom deployment output's fileame
```
Comment (hus0) [inline doc/molior-deploy.8.txt]: Fix typo.

File: doc/molior-project-manager.1.txt [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ molior-project-manager [OPTIONS] PROJECT VERSION OPERATION [ARGS]...
+
+DESCRIPTION
+ molior-project-manager clones the git repos of a molior project and is able to modify all of them with
+ simpe operations.
+
+OPTIONS
+ -m MOLIOR_URL      Alternative molior url (Default in ~/.moliorrc)
+ -x PATTERN         Exclude pattern (one ore multiple)
+ -a                 Add a new version instead of overwriteing the old one
```
Comment (hus0) [inline doc/molior-project-manager.1.txt]: Fix typo.

**Review by hus0 — APPROVED** [review]
This is really nice! Maybe check manpages againts spellchecker.

File: doc/molior-project-manager.1.txt (hunk 3 above)
Comment (neolynx) [reply doc/molior-project-manager.1.txt]: thanks, done!

File: doc/molior-deploy.8.txt (hunk 2 above)
Comment (neolynx) [reply doc/molior-deploy.8.txt]: which one ?

**Review by szakalboss — APPROVED** [review]

File: doc/molior-deploy.8.txt (hunk 2 above)
Comment (hus0) [reply doc/molior-deploy.8.txt]: fileame -> file**n**ame

**Pull request merged**